Scripts/RestartMenu.gd

```diff
@@ -0,0 +1,14 @@
+extends CanvasLayer
+
+func _ready() -> void:
+	hide()
+	GameManager.connect("gameOver", self, "gameOver")
+
+func gameOver(playerkilled: String) -> void:
+	print("playerkilled -> ", playerkilled)
```

Comment:
Is this really needed?
It looks like a leftover from a test phase 😅

Reply:
true